ccloud/docs/index.rst

```diff
@@ -45,13 +55,25 @@ Run demo
 
      $ cd examples/ccloud
 
-3. Start the entire demo by running a single command that brings up the local self-hosted Confluent Platform using `confluent cli, Confluent Replicator, and the KSQL streaming application. This will take less than 5 minutes to complete.
+3. By default, the demo runs with a locally running |sr|. If you prefer to use Confluent Cloud |sr| instead:
+
+   .. figure:: images/schema-registry-cloud.jpg
+       :alt: image
+
+   a. `Enable <http://docs.confluent.io/current/quickstart/cloud-quickstart.html#step-3-configure-sr-ccloud>`__ Confluent Cloud |sr| prior to running the demo
+   b.  Modify the appropriate start script (see next step) and set `USE_CONFLUENT_CLOUD_SCHEMA_REGISTRY=true`
```

Comment:
@ybyzek I think it would be great to mention that the schema-registry status (docker-compose ps) will be Exit 137 if a user uses Confluent Cloud Schema Registry.

Reply:
@NathanNam good suggestion.  Now the following message is printed in that scenario:

```
Killing the local schema-registry Docker container to use Confluent Cloud Schema Registry instead
Killing schema-registry ... done
```